mainapp/models.py

```diff
@@ -91,6 +94,8 @@ class Request(models.Model):
         default = 'new'
     )
     supply_details = models.CharField(max_length=100, blank=True)
+    additional_phone_numbers = ArrayField(models.CharField(max_length=200), blank=True, default=[],
```

Comment:
Isn't it [recommended](https://docs.djangoproject.com/en/2.1/ref/contrib/postgres/fields/#django.contrib.postgres.fields.ArrayField) to use `list` instead of `[]` as `default` for `ArrayField` ?

Reply:
Updated, please review